testing/test_sampler.py

```diff
@@ -18,69 +17,69 @@ def _cmpt_label(ds_loader):
     cmpt_label, cmpt_sample = {0: 0, 1: 0}, 0
     for i, batch in enumerate(ds_loader):
         for gt in batch['gt']:
-            for idx in range(len(gt)):
-                if np.any(gt[idx].numpy()[0]):
-                    cmpt_label[1] += 1
-                else:
-                    cmpt_label[0] += 1
-                cmpt_sample += 1
+            if np.any(gt.numpy()):
```

Comment:
I am pretty sure it is minor, but this test failed locally on my computer: 
![image](https://user-images.githubusercontent.com/49137243/83652800-07e87980-a589-11ea-83b0-0df8c39d0586.png)

We should probably modify this assertion (change the hard limit of 10) or there might be an error during the sampling?

Reply:
Oh yep!
A conjunction of not a lot of slices (1 subject) + large batch_size --> no strict 50/50
I am going to fix this